src/utils/createUrl.js

```diff
@@ -0,0 +1,21 @@
+const parseQueryParams = (params = {}) => Object.entries(params)
```

Comment:
I do not think it is parsing really. It would be if you parsed the string and generated query params object. In this case it rather `stringifyQueryParams`.